json_schemas/auth_users.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "API Updater Schema",
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "email_address": {
+      "type": "string",
+      "pattern": "^[^@^\\s]+@[^@^\\.^\\s]+(\\.[^@^\\.^\\s]+)+$"
```

Comment:
Not sure how to implement this, changes the way validation works as it all shares a validate method which would take differeing params, and didn't work when I tried it. Bit short of time right now so will park this into a new story to implement formats on validations.